pages/index.rst

```diff
@@ -68,10 +68,17 @@
    *The LBNL Physics Division is supported by the U.S. Department of
    Energy Office of Science High Energy Physics.*
 
-   *The Cerro Tololo Inter-American Observatory and the National
-   Optical Astronomy Observatory are operated by the Association
-   of Universities for Research in Astronomy (AURA) under cooperative
-   agreement with the National Science Foundation.*
+   *NSF's NOIRLab (National Optical-Infrared Astronomy Research
+   Laboratory), the US center for ground-based optical astronomy,
+   operates the international Gemini Observatory (a facility of NSF
+   NRC-Canada, ANID-Chile, MCTIC-Brazil, MINCyT-Argentina, and
+   KASI-Republic of Korea), Kitt Peak National Observatory (KPNO),
+   Cerro Tololo Inter-American Observatory (CTIO), the Community
+   Science and Data Center (CSDC), and Vera C. Rubin Observatory (in
+   cooperation with DOE's SLAC National Accelerator Laboratory). It is
+   managed by the Association of Universities for Research in Astronomy
+   (AURA) under a cooperative agreement with NSF and is headquartered
+   in Tucson, Arizona*
```

Comment:
I assume there should be a period at the end of this sentence?

Reply:
Oops, sorry, yes, thanks. If you want to check against the actual language I was sent, here's a copy/paste:

```
[NSF’s NOIRLab](https://noirlab.edu/) (National Optical-Infrared Astronomy Research Laboratory), the US center for ground-based optical-infrared astronomy, operates the international [Gemini Observatory](https://www.noirlab.edu/public/programs/gemini-observatory/) (a facility of [NSF](https://www.nsf.gov/), [NRC–Canada](http://www.nrc-cnrc.gc.ca/eng/solutions/facilities/gemini.html), [ANID–Chile](http://www.conicyt.cl/astronomia/oficina-gemini-chile/), [MCTIC–Brazil](https://www.gov.br/mcti/pt-br), [MINCyT–Argentina](http://www.geminiargentina.mincyt.gob.ar/), and [KASI–Republic of Korea](http://kgmt.kasi.re.kr/kgmtscience)), Kitt Peak National Observatory ([KPNO](https://www.noirlab.edu/public/programs/kitt-peak-national-observatory/)), Cerro Tololo Inter-American Observatory ([CTIO](https://www.noirlab.edu/public/programs/ctio/)), the Community Science and Data Center ([CSDC](https://www.noirlab.edu/public/programs/csdc/)), and [Vera C. Rubin Observatory](https://www.noirlab.edu/public/programs/vera-c-rubin-observatory/) (in cooperation with [DOE](https://www.energy.gov/science/office-science)’s [SLAC](https://www6.slac.stanford.edu/) National Accelerator Laboratory). It is managed by the Association of Universities for Research in Astronomy ([AURA](https://www.aura-astronomy.org/)) under a cooperative agreement with [NSF](https://www.nsf.gov/) and is headquartered in Tucson, Arizona.
```

I decided to drop the urls as links aren't included elsewhere in the boilerplate and to make maintenance easier.

Reply:
(I have no other planned changes).

Reply:
Yikes, I found another difference: "ground-based optical-infrared astronomy" somehow got changed to "ground-based optical astronomy". I'm going go through word-for-word.